Automattic/jetpack
# VideoPress: Limit file extensions based on initial state allowed extensions

Fixes #26541

#### Changes proposed in this Pull Request:

* Removes the hardcoded extensions and use the allowed extensions provided in the initial state on both upload buttons

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard and click on a buton to upload a video
* Check that the file selection now has specific extensions by default

![2022-10-14_15-42-50](https://user-images.githubusercontent.com/8486249/195938102-10a3dc73-d9d2-4729-89c6-0ebd0e655ea9.png)
